device/arista/x86_64-arista_7170_64c/Arista-7170-64C/buffers_defaults_t0.j2

```diff
@@ -10,37 +14,70 @@
 {%- macro generate_buffer_pool_and_profiles() %}
     "BUFFER_POOL": {
         "ingress_lossless_pool": {
-            "size": "33329088",
+            "size": "{{ ingress_lossless_pool_size }}",
```

Comment:
the ingress lossless pool size is changed from 32M -> 4M, why such change. I do not understand the rationale behind this change.